K8s/helm/values.yaml

```
@@ -22,8 +22,12 @@ eum:
 psp:
   enabled: false
 
-# For the mini ones minikube, minishift set to true
-nodeport: false
+
+web:
+  # autotrace-webhook binaries to log in debug level
+  debug_autotrace: false
+  # For the mini ones minikube, minishift set to true
+  nodeport: false
```

```suggestion
  nodeport: 0
```

Rather than a boolean I would make it a "optional" int value so the port can be adjusted to whatever the user wants. With a 0 value the nodePort would be disabled. Also Helm discourages heavy nesting. I'd probably opt for `webDebugAutotrace` and `webNodePort`. If we wanted dynamic assignment by the cluster of a nodePort then it might make sense to have a boolean and a numeric but it's probably over-engineering for a demo app.